docs/manual/docs/cmd/graph/teams/teams-channel-list.md

```diff
@@ -0,0 +1,36 @@
+# graph teams channel list
+
+List the channels in a specified Microsoft Teams team
+
+## Usage
+```sh
+graph teams channel list [options]
+```
+
+## Options
+
+Option|Description
+------|-----------
+`--help`| output usage information
+` -i, --groupId <groupId> `| The ID of the group to list the channels
+` -o, --output [output] `| Output type. json|text. Default text
```

Comment:
json|text should be marked as code, otherwise they will break the table rendering, because | is a column indicator